event-management-frontend/src/components/userEventDetailsPage/CarouselSlide.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { Grid, CardMedia } from "@material-ui/core";
+import Arrow from "./Arrow";
+import { EventImage } from "../../model/EventImage";
+import { useStyles } from "../../styles/CarouselSlideStyles";
+
+interface Props {
+  images: EventImage[];
+}
+
+const testImages = [{ url: "https://i.ibb.co/KNwnXRj/no-image.jpg" }];
```

Comment:
Probably it's better to save this placeholder image, add it in the local `assets` folder and use it from there.